example/README.md

```diff
@@ -2,27 +2,28 @@
 
 ## Overview
 
-The example shows one of the important Registry use-cases — services advertisement and discovery. So it is about advertisement by providers, discovery and usage by clients with the same Aqua code transparently without any knowledge about particular peer and service ids.
+The example shows one of the important Registry use-cases — services advertisement and discovery. So it is about advertisement, discovery and usage by clients with the same Aqua code transparently without any knowledge about particular peer and service ids.
 
 In the beginning, we will deploy a Rust echo service and call it with the Fluence CLI.
 Then we will start a JS/TS client with echo service and run it.
 
-Finally, we will use Registry to call different types of services with exactly one piece of code.
+Secondly, we will use Registry to call different types of services with exactly one piece of code.
+And finally, we will show how to remove service records.
 
 ## Requirements
 
 ```markdown
 node: >= 16
 rust: rustc 1.63.0-nightly
-@fluencelabs/cli: 0.2.7
+@fluencelabs/cli: 0.2.13
 ```
 
 # Set up environment
 
 1. Install `fluence` cli:
 
     ```bash
-    npm i -g @fluencelabs/cli@0.2.7
+    npm i -g @fluencelabs/cli@0.2.13
```

Comment:
that's risky. as we have seen with the bad version management upstream.

Reply:
maybe we should add somewhere in the readme that if any of the fluence cli commands fail due to some version mismatch to use the `fluence dependency` command to manually specify the correct version?